packages/features/calendars/DatePicker.tsx

```diff
@@ -289,4 +301,41 @@ const DatePicker = ({
   );
 };
 
+/**
+ * Takes care of selecting a valid date in the month if the selected date is not available in the month
+ */
+const useHandleInitialDateSelection = ({
+  daysToRenderForTheMonth,
+  selected,
+  onChange,
+}: {
+  daysToRenderForTheMonth: { day: Dayjs | null; disabled: boolean }[];
+  selected: Dayjs | Dayjs[] | null | undefined;
+  onChange: (date: Dayjs | null) => void;
+}) => {
+  // Let's not do something for now in case of multiple selected dates as behaviour is unclear and it's not needed at the moment
+  if (selected instanceof Array) {
+    return;
```

Comment:
instanceof doesn't work properly in iframes. can we use Array.isArray() here?

Reply:
I don't think there should be an issue wrt iframes. Such an issue might happen if we do some cross iframe data passing, I guess. But here selected and Array are always of same window.